src/compat/fkeylist.c

```diff
@@ -46,7 +46,7 @@ ts_relation_get_fk_list(Relation relation)
 		if (constraint->contype != CONSTRAINT_FOREIGN)
 			continue;
 		info = (ForeignKeyCacheInfoCompat *) newNode(sizeof(ForeignKeyCacheInfoCompat),
-													 T_ForeignKeyCacheInfoCompat);
+													 T_ForeignKeyCacheInfo);
```

Comment:
AFAIK, this file isn't even compiled with PG11+, so I think this is a false positive by clang-tidy

Reply:
Yeah, was overly eager in including files. Looked like a low-hanging fruit.